### PR TITLE
Some text updates to intro section

### PR DESCRIPTION
**Pull request by rmglennon**
Filled in the to do items on API key, bugs/issues, service limits from @kdiluca. (This is still in progress, just doing an initial check in.)

### CODE REVIEW

**Comment by dnesbitt61** [comment]
:shipit:

File: docs/valhalla_service.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-* Bicycle routes have a limit of TTT kilometers.
-* Automobile routes have a limit of UUU kilometers.
-* No more than 2 locations can be provided.
+* Pedestrian routes have a limit of 100 kilometers.
+* Bicycle routes have a limit of 500 kilometers.
+* Automobile routes have a limit of 5,000 kilometers.
+* No more than two locations can be provided.
 
-We hope to relax the limits on route distance as we improve performance on longer routes. The limit of number of locations will be increased shortly, stay tuned.
+Send a message if you need higher limits in the meantime.
```
Comment (kevinkreiser) [inline docs/valhalla_service.md]: might want to reiterate where you want the message to be sent

File: docs/valhalla_service.md [hunk 2]
```diff
@@ -64,11 +64,11 @@ Valhalla uses dynamic, run-time costing to form the route path. The route reques
 Costing models currently supported include:
 
 * costing = auto. Standard costing for driving routes (using car, motorcycle, truck, etc.) that obeys automobile driving rules (access, turn restrictions, etc.) that provides a short time path (not guaranteed to be shortest time) that also uses intersection costing to help minimize turns and maneuvers or road name changes. Routes also tend to favor highways (higher classification roads: motorways, trunk).
-* costing = auto_shorter. Alternate costing for driving that is intended to provide a short 
+* costing = auto_shorter. Alternate costing for driving that is intended to provide a short
 *  path (though not guaranteed to be shortest distance) that obeys driving rules (access, turn restr)
```
Comment (kevinkreiser) [inline docs/valhalla_service.md]: turn restr? should probably spell out restrictions

**Comment by kevinkreiser** [comment]
looks good

**Comment by rmglennon** [comment]
Thanks @kevinkreiser (and @dnesbitt61 for the merge). Will look at these notes and the restrictions section (hadn't gotten through that part yet!). I took out some of the intro stuff and will put it elsewhere, too.

File: docs/valhalla_service.md (hunk 2 above)
Comment (dnesbitt61) [inline docs/valhalla_service.md]: yea, this is an error in my initial entry

**Comment by kdiluca** [comment]
It would probably be nice to show an example of a simple request.  I have this one saved for a route that avoids Canada by using the country_crossing_penalty:

http://valhalla.api.dev.mapzen.com/route?json={"locations":[{"lat":42.358528,"lon":-83.271400,"street":"Appleton"},{"lat":42.996613,"lon":-78.749855,"street":"Ranch Trail"}],"costing": "auto","costing_options":{"auto":{"country_crossing_penalty":2000.0}},"directions_options":{"units":"miles"}}&api_key=<mapzen developer key>

File: docs/valhalla_service.md (hunk 1 above)
Comment (rmglennon) [inline docs/valhalla_service.md]: Thinking this should be an email, rather than a GitHub issue...will add the address here.

**Comment by rmglennon** [comment]
Thanks, @kdiluca! There is a section on sample requests. I added this one (will need to be updated without the dev URL).

**Comment by rmglennon** [comment]
OK, think this addresses all the "to do" items, as well as updated formatting and some rewrites. Take a look and see if these changes can be merged. Thanks!

**Comment by kevinkreiser** [comment]
we dont currently support pbf output but it is planned. might want to also name this out_format to match the other parameter names. i would also favor the values without the '.' for this. anyway, this is sweet. SHIP SHIP SHIP

**Comment by kevinkreiser** [comment]
:shipit:

**Pull request merged**